MapboxNavigation/PassiveLocationManager.swift

```diff
@@ -48,7 +48,7 @@ open class PassiveLocationManager: NSObject, MGLLocationManager {
         // CLLocationManager.accuracyAuthorization was introduced in the iOS 14 SDK in Xcode 12, so Xcode 11 doesn’t recognize it.
         guard let accuracyAuthorizationValue = dataSource.systemLocationManager.value(forKey: "accuracyAuthorization") as? Int,
               let accuracyAuthorization = MBNavigationAccuracyAuthorization(rawValue: accuracyAuthorizationValue) else {
-            return MBNavigationAccuracyAuthorization(rawValue: 0)!
+            return .fullAccuracy
```

Comment:
Because we use the result from the key-value call and check the `.reducedAccuracy` only. Maybe it's not necessary to make the change in the `PassiveLocationManager`

Reply:
The change doesn't affect my code at all. Minh and I noticed the use of forced downcasting and realized that it could be replaced with `.fullAccuracy` to make this return safer.